cg/meta/qc_metrics/collect_qc_metrics.py

```diff
@@ -83,11 +86,24 @@ def create_qc_metrics_request(self, case_id: str) -> CreateQCMetricsRequest:
             files=file_paths_and_tags,
         )
 
-    def get_qc_metrics(self, case_id) -> dict:
+    def get_case_qc_metrics(self, case_id) -> dict:
```

Comment:
```suggestion
    def get_case_qc_metrics(self, case_id: str) -> dict:
```
Just because we have both internal_ids and ids.